curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8c465fa7b0c252f4a8f0c.md

```diff
@@ -17,6 +19,8 @@ const total = document.querySelector('#total');
 total.innerText = value;
 ```
 
+**Note:** You don't need to add more than the third line in the example above.
```

Comment:
Maybe we could go with this wording

"Your answer should only be two lines of code"

or something similar to that.

Reply:
Ok.

I will get this done when I get back to work.

Reply:
That sounds good to me too, @jdwilkin4.